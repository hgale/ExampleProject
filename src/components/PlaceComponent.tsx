import * as React from 'react';
import {  StyleSheet, Text, View} from 'react-native';
import { colors } from 'src/utils/styles';

interface Props {
  description: string;
}

const PlaceComponent = (props: Props) => (
  <View style={ styles.place }>
    <Text style={ styles.placeText }> { props.description } </Text>
  </View>
);

const styles = StyleSheet.create({
  place : {
    width: '80%',
    height: 60,
    justifyContent: 'center',
    alignItems: 'center',
    marginTop: 10,
    marginBottom: 10,
  },
  placeText: {
    color: colors.WHITE,
    alignSelf: 'flex-start',
  },
});

export default PlaceComponent;
