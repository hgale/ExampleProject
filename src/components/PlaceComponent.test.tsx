import * as React from 'react';
import 'react-native';
import * as renderer from 'react-test-renderer';

import PlaceComponent from 'src/components/PlaceComponent';

it('renders correctly', () => {
  const component = renderer.create(<PlaceComponent description='House of Shields' />);
  expect(component).toMatchSnapshot();
});
