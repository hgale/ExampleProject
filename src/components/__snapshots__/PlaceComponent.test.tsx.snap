// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": 60,
      "justifyContent": "center",
      "marginBottom": 10,
      "marginTop": 10,
      "width": "80%",
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "flex-start",
        "color": "rgb(255, 255, 255)",
      }
    }
  >
     
    House of Shields
     
  </Text>
</View>
`;
